toolkit/tools/imagegen/configuration/parse_partition.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Utility to create and manipulate disks and partitions
+
+package configuration
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+
+	"microsoft.com/pkggen/internal/logger"
+)
+
+// In kickstart installation, the kickstart file will include a file (/tmp/part-include)
```

Comment:
Documentation for exported functions (ones starting with a capital letter) should be written in a form of  "FuncName does x y z ..." (e.g. "ParseKickStartPartitionScheme parses file partitionFile extracting values into config"))

Reply:
Fixed